uiowa/uiowa
# Classrooms - search page enhancements

Resolves #6434.

# To do:

- [ ] ~Get mutliselect working for building filter~ - will be fixed in follow-up issue
- [x] See if error with type filter is fixable so that checkboxes can be used
- [x] Move sort options out of filter area
- [x] Fix issue where select list options are overlapping/hidden behind cards
- [x] Check set-up of room/tech features - should rooms only display if they meet all selections, or any?

# How to test:
```
ddev blt frontend && ddev blt ds --site=classrooms.uiowa.edu && ddev drush @classrooms.local uli find-university-classroom
```


- checkout & pull branch, `ddev blt ds --site=classrooms.uiowa.edu`, `uli` into site
- go to https://classrooms.uiowa.ddev.site/find-university-classroom
- test new sort options (building asc & desc, max capacity asc & desc) work as expected
- test multiselect filters work (type, technology, features)
- Check https://classrooms.uiowa.ddev.site/programmed-classrooms was not affected by these changes
